README.md

```diff
@@ -1,2 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # GrowMeghaPortfolio
-Portfolio Website
+- Made portfolio website home page using flexbox.
+
+
+## Built With
+
+- Major languages
+    - Html
+    - CSS
+    - flexbox
+    - Node
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Vistual Studio Code
+- Browser (Firefox / Google Chrome)
+- Node.js (v6.11.2+)
+- linters 
+
+### Setup
+1. Install Vistual Studio Code from https://code.visualstudio.com/
+2. Install Live Server VS plugin from https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer
+3. Open a project and click to Go Live from the status bar to turn the server on/off.
+
+### Usage
+Showcase your portfolio.
+
+## Authors
+
+👤 **Megha Bodke**
+
+- GitHub: [@megha-n-bodke](https://github.com/megha-n-bodke)
+- LinkedIn: [Megha Bodke](https://www.linkedin.com/in/megha-bodke/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to Microverse for giving this opportunity to learn ...
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great job using the Microverse Template here 💯👌 and I think it would be better to add the `MIT.md` file, you can use [this template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and don't forget to customize it 👌.

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.